doc/module-developer/src/hvac-network/interfacing-with-plant.tex

```diff
@@ -44,9 +44,7 @@ \subsection{Sizing}\label{sizing}
 \item
   PlantFinalSizesOkayToReport This public boolean flag is declared in DataPlant. It starts out false and is set to true when all the sizing is done and it is okay to report the final size values. Component models should delay reporting final sizes until this is set to true.
 \item
-  InitLoopEquip This logical flag is passed as an argument with SimPlantEquip and carries through to the main simulation calls for component models. This argument is set to TRUE when the plant sizing is being conducted. The component model needs to handle this argument such that when TRUE the get input, initialization and sizing routines are run, but the calculation routine does not. When TRUE most supply side component models need to return values for the minimum, maximum, and optimal capacities (in terms of loop loads that the device can meet). For plant components with more than one connection to a plant loop, a leading loop connection must be determined and the component sizing routine called with InitLoopEquip is called for only that plant loop. For a example, a chiller only calls its sizing routine when called from a the chilled water loop and does not call it sizing routine when called from the condenser loop.
```

Comment:
I removed this whole item, since it is not a boolean flag anymore. Should I keep some of this information somewhere else?

Reply:
Well, I guess the logic from the InitLoopEquip flag handling is now a part of the component's initLoopEquip member method logic.  So maybe this should be in the PlantComponent.hh file as a comment in the PlantComponent::initLoopEquip method?

Reply:
I will move it there. Thank you for the suggestion ✌🏽